internal/juno/juno.go

```diff
@@ -133,24 +138,29 @@ func (n *Node) Run() error {
 	}
 	n.blockManager = block.NewManager(blockDb)
 
-	n.feederClient = feeder.NewClient(n.cfg.Network.URL(), feederGatewaySuffix, nil)
-	n.virtualMachine = cairovm.New(n.stateManager)
-	n.synchronizer = syncer.NewSynchronizer(n.cfg.Network, n.cfg.EthNode, n.feederClient,
-		n.syncManager, n.stateManager, n.blockManager, n.transactionManager)
+	n.feederClient, err = feeder.NewClient(n.cfg.Network.URL(), feederGatewaySuffix, nil, n.logger.Named("FEEDER"))
+	if err != nil {
+		return err
+	}
+	n.virtualMachine = cairovm.New(n.stateManager, n.logger.Named("VM"))
+	n.synchronizer, err = syncer.NewSynchronizer(n.cfg.Network, n.cfg.EthNode, n.feederClient,
+		n.syncManager, n.stateManager, n.blockManager, n.transactionManager, n.logger.Named("SYNC"))
 
-	jsonRpc, err := starkNetJsonRPC(n.stateManager, n.blockManager, n.transactionManager,
-		n.synchronizer, n.virtualMachine)
 	if err != nil {
 		return err
 	}
-	rpcAddr := ":" + strconv.FormatUint(uint64(n.cfg.RpcPort), 10)
-	n.rpcServer, err = rpc.NewHttpRpc(rpcAddr, rpcSuffix, jsonRpc)
+
+	rpcLogger := n.logger.Named("RPC")
+	jsonRpc, err := starkNetJsonRPC(n.stateManager, n.blockManager, n.transactionManager,
+		n.synchronizer, n.virtualMachine, rpcLogger)
```

Comment:
We are using the `rpcLogger` in two places: the call to `starknetJsonRPC` and when we set up the RPC server on line 160. Since they are both RPC-related, I thought it made sense to use the same logger.

If that's a bad idea or if there is a better way to do this, please let me know.